modules/apps/layout/layout-type-controller/layout-type-controller-test/src/testIntegration/java/com/liferay/layout/type/controller/asset/display/internal/portlet/test/AssetDisplayPageFriendlyURLProviderImplTest.java

```diff
@@ -124,6 +128,27 @@ public void testGetFriendlyURLWithConfiguredURLSeparator()
 		}
 	}
 
+	@Test
+	public void testGetFriendlyURLWithoutDisplayPageEntry() throws Exception {
+		Layout layout = LayoutTestUtil.addTypePortletLayout(
+			_group.getGroupId());
+
+		_setUpThemeDisplay(layout);
+
+		JournalArticle journalArticle = JournalTestUtil.addArticle(
+			_group.getGroupId(),
+			JournalFolderConstants.DEFAULT_PARENT_FOLDER_ID,
+			JournalArticleConstants.CLASS_NAME_ID_DEFAULT, StringPool.BLANK,
+			true, RandomTestUtil.randomLocaleStringMap(),
+			RandomTestUtil.randomLocaleStringMap(),
+			RandomTestUtil.randomLocaleStringMap(), layout.getUuid(),
+			LocaleUtil.getSiteDefault(), null, false, false,
+			ServiceContextThreadLocal.getServiceContext());
```

Comment:
It can be replaced by:
```java
JournalTestUtil.addArticle(
	_group.getGroupId(),
	JournalFolderConstants.DEFAULT_PARENT_FOLDER_ID);
```

Also the call in `setUp` method.

Reply:
I can't use that version; I need to set the layoutUUID. Also, it's a local variable only used once. If there will be another use case for it, we can move it to the setup, but currently, I don't think it is necessary.